tests/model_fixtures.py

```diff
@@ -1,27 +1,43 @@
 """
 This is where we put all the data about creating / updating models
 """
-from dataclasses import dataclass
-from typing import Dict, Sequence, Union
+from dataclasses import dataclass, field
+from typing import Dict, List, Sequence, Union
 
 APIOpts = Dict[str, Union[int, str]]
 
 
 @dataclass
-class ModelFixtures:
+class ModelFixture:
     path: str
     required_fields: Sequence[str]
     create_opts: Sequence[APIOpts]  # valid ways to create the model
     invalid_opts: Sequence[APIOpts]  # setting these on creation or update should error
     update_opts: Sequence[APIOpts]  # setting these on update should be valid
 
 
+@dataclass
+class ModelRelationFixture:
+    field_name: str  # Name of the field cooresponding to the related table
```

Comment:
```suggestion
    field_name: str  # Name of the field corresponding to the related table
```

I would also like a one-line comment about what this class does.

Reply:
>  a one-line comment about what this class does.

Ended up being two lines; hopefully this makes sense https://github.com/pangeo-forge/pangeo-forge-orchestrator/pull/31/commits/45c5004fb47fe678898fa47eb47f653e263668c8